ui/src/album/AlbumListView.js

```diff
@@ -89,14 +90,16 @@ const AlbumListView = ({ hasShow, hasEdit, hasList, ...rest }) => {
       {isDesktop && <DurationField source="duration" />}
       <AlbumContextMenu
         source={'starred'}
-        sortBy={'starred ASC, starredAt ASC'}
+        sortBy={config.enableFavourites && 'starred ASC, starredAt ASC'}
```

Comment:
Instead of messing with the `sortBy` field (and in this case breaking it), you should just disable the `sortable` property for this column. See https://github.com/navidrome/navidrome/blob/ac37bf3631610747001f44fe843517f3fdf0ac08/ui/src/album/AlbumSongs.js#L123